crates/bindings-csharp/Runtime/LogStopwatch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpacetimeDB
+{
+    public class LogStopwatch
```

Comment:
Can't look up the spec right now, but I believe we agreed that timers should also be IDisposable (ending the timer at the end of the using scope).

Reply:
I was back-and-forth on this, but I guess there's no harm in adding it